services/brig/brig.yaml.example

```diff
@@ -72,6 +72,7 @@ turn:
 
 optSettings:
   setActivationTimeout: 5
+  setTeamInvitationTimeout: 10
```

Comment:
Doesn't 5 seconds suffice for testing purposes?

Reply:
> Doesn't 5 seconds suffice for testing purposes?
 
👍